.github/workflows/conformance.yaml

```diff
@@ -17,12 +17,21 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
+      - name: Cache Go Modules
+        uses: actions/cache@v4
         with:
-          fetch-depth: 1
+          path: |
+            ~/.cache/go-build
+            ~/go/pkg/mod
+          key: ${{ runner.os }}-gomod-conformance-${{ hashFiles('gradle.properties', 'gradle/libs.versions.toml') }}
+          restore-keys:
+            ${{ runner.os }}-gomod-conformance-
```

Comment:
Conformance workflow uses a different cache as it also downloads and builds the conformance binary from bufbuild/protovalidate.